mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -3539,6 +3539,45 @@ def _get_trace_status_from_root_span(self, spans: list[Span]) -> str | None:
                     return TraceState.OK.value
         return None
 
+    def get_trace(self, trace_id: str, allow_partial: bool = False) -> Trace | None:
+        if allow_partial is False:
+            retry_count = 0
+            while retry_count < 3:
```

Comment:
```suggestion
            for _ in range(3):
```

can we use for loop?